src/sites/signalen/pages/SignalenRootPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { Screen } from '@amsterdam/design-system-react'
+import { Outlet } from 'react-router-dom'
+import { SiteFooter } from '../components/SiteFooter'
+import { SiteHeader } from '../components/SiteHeader'
+import '../components/signalen.css'
+
+export const SignalenRootPage = () => (
+  <>
+    <Screen maxWidth="wide">
+      <SiteHeader />
+      <main id="main">
```

Comment:
If you leave out the `SkipLink`, you can also remove the `id` here and the enclosing `<></>`. Or restore the Skip Link?

Reply:
I originally removed it, because the current Signalen doesn't have a menu (so you don't need a skiplink). The new design does have a menu though, so I'll put it back

Reply:
In this or a new PR?

Reply:
In this one, it's back already

Reply:
Ah, found it in the Header.